README.md

```diff
@@ -80,7 +80,8 @@ This approach runs the LiteFarm applications and database server directly on you
        exit;       
        ```
 
-3. In a terminal, run `npm i` under the root folder of the repository, `npm i` under `packages/api`, and `pnpm i`
+3. In a terminal, run `npm i pnpm -g` *if pnpm is not already installed*, run `npm i` under the root folder of the
```

Comment:
To ease automation, please put all dependencies in package.json or equivalent (https://12factor.net/dependencies). Please avoid global installs if not necessary.